justfile

```diff
@@ -112,6 +112,10 @@ expose-ports:
   CLOUDBEAT_POD=$( kubectl get pods -o=name -n kube-system | grep -m 1 "cloudbeat" ) && \
   kubectl port-forward $CLOUDBEAT_POD -n kube-system 40000:40000 8080:8080
 
+generate-mocks:
```

Comment:
Nice! can you please document this? would be nice to mention this [here](https://github.com/elastic/cloudbeat/blob/main/dev-docs/Development.md#mocking)